pom.xml

```diff
@@ -101,7 +101,8 @@ under the License.
 		<chill.version>0.7.4</chill.version>
 		<asm.version>5.0.4</asm.version>
 		<zookeeper.version>3.4.6</zookeeper.version>
-		<curator.version>2.12.0</curator.version>
+		<curator.version>2.11.0</curator.version>
```

Comment:
Oh..My environment was not able to get 2.12.0. So to make things compile I included this change. Will revert it.